extra/exports

```diff
@@ -101,7 +101,7 @@ box_replace
 box_return_mp
 box_return_tuple
 box_schema_version
-box_select
```

Comment:
Shouldn't we keep the old version for compatibility?

Reply:
`box_select` isn't a part of the public API so I guess not.

Reply:
It is present here:
https://github.com/tarantool/tarantool/blob/ca58d6c92a10c4ab80f07425c51f20225e5611e8/static-build/test/exports.test.lua#L104-L119

Reply:
Maybe by mistake, but we need to check if it's actually used by any of our modules (e.g. tdg).

Reply:
We've changed `box_select` signature (added arguments for pagination), so the compatibility with old `box_select` is already broken.

Reply:
I've searched for its usage across `tarantool` organization - it is not used outside the core part.

Reply:
Agree with @locker here.

Reply:
AFAIK, `box_select` was in public API until it became to use `port`.